components/CryptoSection.tsx

```diff
@@ -0,0 +1,139 @@
+import { useCallback, useMemo, useState } from "react";
+import React from "react";
+import Web3 from "web3";
+import { AbiItem } from "web3-utils";
+import verifyAccount from "web3/utils/verifyAccount";
+
+import showToast from "@lib/notification";
+
+import { Button } from "@components/ui/Button";
+
+import { useContracts } from "../contexts/contractsContext";
+import genericAbi from "../web3/abis/abiWithGetBalance.json";
+
+interface Window {
+  ethereum: any;
```

Comment:
There are ways to type this. `web3` (or `ethers`) library should have support for this.